shapes/Circle.cpp

```diff
@@ -48,12 +49,6 @@ std::string Circle::toString() const
            std::to_string(position.x()) + ", " +
```

Comment:
```suggestion
           std::to_string(position.x()) + jconstants::DIMENSION_INTERRUPTOR + ' ' +
```
Try this for added style and modifiability.  Same for all `toString()` methods